From 3f72d8036ad8e7bc58dae0a479be4133c9b2f3eb Mon Sep 17 00:00:00 2001
From: Paul Keith <javelinanddart@gmail.com>
Date: Fri, 6 May 2022 10:41:52 +0530
Subject: [PATCH 03/10] fixup! libstagefright: omx: Add support for loading
 prebuilt ddp and ac4 decoder lib

[HELLBOY017]: Also allow to load dolby codec in MediaCodecList

Change-Id: I3b85c15889c9e3aef152e726a83936234103254e
---
 media/libstagefright/MediaCodecList.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/media/libstagefright/MediaCodecList.cpp b/media/libstagefright/MediaCodecList.cpp
index 4ad3276743..03bf12e6dc 100644
--- a/media/libstagefright/MediaCodecList.cpp
+++ b/media/libstagefright/MediaCodecList.cpp
@@ -319,6 +319,7 @@ const sp<AMessage> MediaCodecList::getGlobalSettings() const {
 //static
 bool MediaCodecList::isSoftwareCodec(const AString &componentName) {
     return componentName.startsWithIgnoreCase("OMX.google.")
+            || componentName.startsWithIgnoreCase("OMX.dolby.")
             || componentName.startsWithIgnoreCase("c2.android.")
             || (!componentName.startsWithIgnoreCase("OMX.")
                     && !componentName.startsWithIgnoreCase("c2."));
-- 
2.34.1

